static/src/javascripts/projects/common/modules/commercial/third-party-tags/plista.js

```diff
@@ -0,0 +1,87 @@
+define([
+    'fastdom',
+    'common/utils/$',
+    'common/utils/config',
+    'common/utils/mediator',
+    'common/utils/detect',
+    'common/utils/template',
+    'common/modules/identity/api',
+    'text!common/views/commercial/plista.html',
+    'lodash/collections/contains'
+], function (fastdom,
+             $,
+             config,
+             mediator,
+             detect,
+             template,
+             identity,
+             plistaTpl,
+             contains) {
+    var containerOuterClassName = 'js-plista',
+        containerInnerClassName = 'js-plista-container';
+
+    return {
+        load: function () {
+            var $containerOuter = $('.' + containerOuterClassName),
+                $containerInner = $('.' + containerInnerClassName);
+
+            fastdom.write(function () {
+                $containerOuter.css('display', 'block');
+                $containerInner.append($.create(template(plistaTpl)));
+
+                this.embed('462925f4f131001fd974bebe', 'innerArticle', 'au', undefined, undefined, containerInnerClassName);
+            }.bind(this));
+        },
+
+        // a modification of the code provided by Plista; altered to be a lazy load rather than during DOM construction
+        embed: function (p, w, g, u, c, containerTag) {
+            var container = document.getElementsByClassName(containerTag)[0],
+                name = 'PLISTA_' + p,
+                lib = window[name];
+            container.innerHTML = '<div data-display="plista_widget_' + w + '"></div>';
+            if (!lib || !lib.publickey) {
+                window[name] = {
+                    publickey: p,
+                    widgets: [{name: w, pre: u}],
+                    geo: g,
+                    categories: c,
+                    dataMode: 'data-display'
+                };
+                require(['js!' + (document.location.protocol === 'https:' ? 'https:' : 'http:') + '//static-au.plista.com/async/' + name + '.js']);
+            } else {
+                lib.widgets.push({name: w, pre: u});
+            }
+        },
+
+        getSection: function () {
+            return config.page.section.toLowerCase().match('news')
+            || contains(['politics', 'world', 'business', 'commentisfree'], config.page.section.toLowerCase()) ? 'sections' : 'all';
+        },
+
+        identityPolicy: function () {
```

Comment:
Repeated logic for turning features on or off should probably live in the commercial-feature-policies file. Each entry there is a policy for a particular domain concept, e.g. 'sensitive content', 'logged in'. Perhaps this should be extracted into there?